doc/glossary.rst

```diff
@@ -77,15 +90,27 @@ Glossary
 
    Pytest
      `Pytest <https://docs.pytest.org/>`_ is a largely used, open-source Python
-     testing tool. Its ``fixture`` mechanism works neatly with ``Ragger``.
+     testing tool. Its ``fixture`` mechanism is integrated into ``Ragger``.
 
    RAPDU
-     "Response APDU", designates the response of an :term:`application`
-     following an :term:`APDU` from the :term:`client` to the application.
+     "Response APDU" (**RAPDU**), designates the response of an
+     :term:`application` following an :term:`APDU` from the :term:`client` to
+     the application.
 
    SDK
      The **SDK** is the open-source code allowing an application to be compiled
```

Comment:
The SDK also provides libraries and other helpful things to write apps. Maybe add a short sentence on that @lpascal-ledger ?

Reply:
Yes indeed!

Reply:
Added this sentence:

> On top of an interface to exploit :term:`BOLOS` capabilities, it provides boilerplate functions, graphic abstractions and other useful libraries for developing apps.